core/domain/feedback_jobs_continuous.py

```diff
@@ -145,6 +145,38 @@ def get_thread_analytics(cls, exploration_id):
         return feedback_domain.FeedbackAnalytics(
             exploration_id, num_open_threads, num_total_threads)
 
+    @classmethod
+    def get_thread_analytics_multi(cls, exploration_ids):
+        realtime_model_ids = cls.get_multi_active_realtime_layer_ids(
+            exploration_ids)
+        realtime_models = cls._get_realtime_datastore_class().get_multi(
+            realtime_model_ids)
+        feedback_thread_analytics_models = (
+            feedback_models.FeedbackAnalyticsModel.get_multi(
+                exploration_ids))
+
+        num_open_threads_list = []
+        num_total_threads_list = []
+        for i in range(len(exploration_ids)):
+            num_open_threads = 0
+            num_total_threads = 0
+            if realtime_models[i] is not None:
```

Comment:
@seanlip Can you please verify this:
- Both models are None - no feedback thread is created and even the MR job has not been run yet. (Yet to write test for this case)
- Realtime Model is None - A new feedback thread is created. MR job is started and then stopped. (Test for this already [exists](https://github.com/oppia/oppia/blob/develop/core/domain/feedback_jobs_continuous_test.py#L515) ?)
- Feedback Analytics model is None - A new feedback thread is created. MR job is started(but never stopped). (Test for this already [exists](https://github.com/oppia/oppia/blob/develop/core/domain/feedback_jobs_continuous_test.py#L327) ?).

Also, I believe that I've addressed all other review comments except this one!